.github/workflows/e2e.yml

```
@@ -31,6 +31,12 @@ jobs:
             echo "is_pr=false" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Add comment to PR
+        id: pr_comment
+        if: steps.check_pr.outputs.is_pr == 'true'
+        run: |
+          gh pr comment "${{ github.event.inputs.pr_or_branch }}" -b "e2e workflow launched on this PR: [View run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

```suggestion
          gh pr comment "${{ github.event.inputs.pr_or_branch }}" -b "e2e workflow launched on this PR: [View run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

Oops. Forgot the close paren for the md link.

---

great catch!

I wish this was easier to test prior to merging ... i fully expect a fixup PR may come after :(

---

I more meant the Mergify part where the workflow gets triggered via the label. That's kind of confusing since I was commenting on the workflow side.

---

fixed the missing paren!